tests/unit/apps/test_auxiliary.py

```diff
@@ -29,14 +29,14 @@
 from tests.unit.apps.utils import add_steps
 
 
-def test_auxiliary_app(status, config, model):
+def test_auxiliary_app(status, config, model, apps_machines):
```

Comment:
non-blocking: I'm not too fond of the name `apps_machines`. Does it contain apps? Machines? If the idea was "here are some machines belonging to some apps" `app_to_machines` would be clearer, imho

Reply:
This will be removed, in another PR where we will inteoduce the couunit and couapplication. At least thats, how I remeber it.